app/models/manageiq/providers/oracle_cloud/cloud_manager/cloud_database.rb

```diff
@@ -166,6 +167,46 @@ def self.create_mysql_database(ext_management_system, options)
     end
   end
 
+  def params_for_update
+    {
+      :fields => [
+        {
+          :component => 'text-field',
+          :id        => 'name',
+          :name      => 'name',
+          :label     => _('Rename Cloud Database'),
+        }
+      ],
+    }
+  end
+
+  def raw_update_cloud_database(options)
+    check_database_type == :oracle ? update_oracle_database(options) : update_mysql_database(options)
+  rescue => err
+    _log.error("cloud database=[#{name}], error: #{err}")
+    raise
+  end
+
+  def update_oracle_database(options)
+    with_provider_connection(:service => 'Database::DatabaseClient') do |connection|
+      connection.update_autonomous_database(
+        ems_ref,
+        OCI::Database::Models::UpdateAutonomousDatabaseDetails.new(
+          :display_name => options[:name]
+        )
+      )
+    end
+  end
```

Comment:
The following settings are needed for this to work (not documented anywhere)
<img width="465" alt="Screen Shot 2022-05-18 at 12 10 42 PM" src="https://user-images.githubusercontent.com/48186199/169091136-7f9f61d7-c664-49fe-8cca-8dd528a32d06.png">
k